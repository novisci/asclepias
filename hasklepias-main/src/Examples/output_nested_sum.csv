concepts,facts,begin,end
"not home",CoinToss,1,10
