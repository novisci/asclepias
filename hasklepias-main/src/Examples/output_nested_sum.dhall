List { concepts : Text, facts : < StayHome | GoOut | CoinToss >, begin : Integer, end : Integer}
