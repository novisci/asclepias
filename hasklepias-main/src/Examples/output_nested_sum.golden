"[[1, 10],{\"getConcepts\":[\"not home\"],\"getFacts\":\"CoinToss\",\"getSource\":null}]"